inst/templates/common/header.mustache

```diff
@@ -22,7 +22,7 @@
 {{#reportMetadata.comparisonSeries}}<strong>Comparison Time Series:</strong> {{reportMetadata.comparisonSeries}}{{/reportMetadata.comparisonSeries}}
 </div>
 <div class="header-col-right">
-<p><strong>Period:</strong> {{reportMetadata.startDate}} to {{reportMetadata.endDate}}</p>
+<p><strong>Period:</strong> {{reportMetadata.displayStartDate}} to {{reportMetadata.displayEndDate}}</p>
```

Comment:
note that just as before my changes, for derivation chain, this appears as just Period: to

if it makes sense to do a check here for whether or not the values are empty, I can try to do that as a part of this task since this didn't take me very long.